client/repoutils.c

```diff
@@ -603,55 +392,50 @@ TDNFRemoveSolvCache(
                   TDNF_SOLVCACHE_DIR_NAME);
     BAIL_ON_TDNF_ERROR(dwError);
 
-    pDir = opendir(pszSolvCacheDir);
-    if(pDir == NULL)
+    TDNFRecursivelyRemoveDir(pszSolvCacheDir);
```

Comment:
we need to get the return from TDNFRecursivelyRemoveDir  to dwError

Reply:
Done